tests/js/server-tests.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+var jqUnit = require("node-jqunit");
+var lunr = require("lunr");
+require("../../index.js");
+fluid.registerNamespace("gpii.tests.devpmt.server");
+
+gpii.tests.devpmt.server.loadTestDataNPSetsTest = function () {
+    var npsets = gpii.devpmt.loadTestDataNPSets("/../../../devpmtTestData/preferences");
+    var haveAlice = false;
+    fluid.each(npsets, function (val) {
+        if (val === "alice") {
+            haveAlice = true;
+        }
+    });
+    jqUnit.assertEquals("Make sure we have alice.", haveAlice, true);
+};
+
+gpii.tests.devpmt.server.loadCommonTermsMetadataTest = function () {
+    var terms = gpii.devpmt.loadCommonTermsMetadata();
+    var fontSize = terms["http://registry.gpii.net/common/fontSize"];
+    jqUnit.assertEquals("Font Size should have a name", "Font Size", fontSize.schema.title);
+};
+
+fluid.defaults("gpii.tests.devpmt.server.caseHolder", {
+    gradeNames: ["fluid.test.testCaseHolder"],
+    modules: [
+        {
+            name: "Devpmt tests...",
+            tests: [
+                {
+                    name: "Devpmt Server Tests",
+                    type: "test",
+                    sequence: [
+                        { funcName: "gpii.tests.devpmt.server.loadCommonTermsMetadataTest" },
```

Comment:
It's not necessary to use the IoC testing framework to sequence simple synchronise jqUnit fixtures - you can just execute them directly